src/Exizent.CaseManagement.Client/CaseManagementApiClient.cs

```diff
@@ -33,6 +35,32 @@ public CaseManagementApiClient(HttpClient httpClient)
         return await GetCaseInternal(caseId, null, options, cancellationToken);
     }
 
+
+    public async Task<EstateItemResponseResourceRepresentation?> PostEstateItem(Guid caseId, EstateItemResourceRepresentationBase estateItem, CancellationToken cancellationToken = default)
```

Comment:
Can we attribute this to show its not fully implimented yet please, same goes for the others